content/litecli-launch.md

```diff
@@ -0,0 +1,37 @@
+Title: "Launching LiteCLI"
+Tags: python, sqlite
+Category: Blog
+Date: 2019-01-03
+
+Today we are happy to announce the launch of [LiteCLI](https://github.com/dbcli/litecli)!
+
+LiteCLI is a user-friendly CommandLine client for SQLite database. 
+
+<img src='https://litecli.com/images/litecli.png' width=750px align=center alt='litecli'/>
+
+
+It is based on the popular [pgcli](https://www.pgcli.com) and [mycli](https://www.mycli.net) projects.
+
+LiteCLI is written in python using the wonderful [prompt-toolkit](https://github.com/prompt-toolkit/python-prompt-toolkit) library. It is
+cross-platform compatible and it is tested on Linux, MacOS and Windows.
+
+It is developed by a team of core devs from around the world. We are friendly
+and welcoming to newcomers. 
+
+The project is lead by [Delgermurun](https://github.com/delgermurun). The
+core team constitutes [Irina Truong](https://github.com/j-bennet),
```

Comment:
Perhaps "the core team includes", or "consists of"?. According to google dictionary, "constitute" means "be (a part) of a whole". So it seems to me that Del & Amjith can constitute a team, but a team cannot constitute Del & Amjith.